dbos/_sys_db.py

```diff
@@ -550,6 +564,7 @@ def get_workflow_status_within_wf(
             {
                 "workflow_uuid": calling_wf,
                 "function_id": calling_wf_fn,
+                "function_name": "getStatus",
```

Comment:
Make it consistent with other names like `DBOS.send`, (all prefix with `DBOS.`...)
```suggestion
                "function_name": "DBOS.get_workflow_status",
```